### PR TITLE
set block gas limit in pool to 7M

### PR DESCRIPTION
**Pull request by tcoratger**
Should close https://github.com/kkrt-labs/kakarot-rpc/issues/1479

### CODE REVIEW

**Review by Eikix — COMMENTED** [review]
can we test this?

File: Makefile [hunk 1]
```diff
@@ -62,6 +62,9 @@ test-ci: load-env
 test-target: load-env
 	cargo test --tests --all-features $(TARGET) -- --nocapture
 
+test-target1: load-env
```
Comment (Eikix) [inline Makefile:65]: why do we add a new target here?

File: Makefile [hunk 2]
```diff
@@ -62,6 +62,9 @@ test-ci: load-env
...
+	cargo test --package kakarot-rpc --test entry --all-features -- tests::mempool::test_maintain_mempool --exact --show-output
+
```
Comment (tcoratger) [inline Makefile:67]:
```suggestion
```

File: Makefile (hunk 1 above)
Comment (tcoratger) [reply Makefile]: not needed, for tests, removed

**Review by greged93 — APPROVED** [review]

**Pull request merged**